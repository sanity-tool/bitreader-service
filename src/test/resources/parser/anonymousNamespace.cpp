namespace {

    static int trivial() {
        return 1;
    }

}
