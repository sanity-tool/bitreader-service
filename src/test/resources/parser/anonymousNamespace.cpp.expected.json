{
  "functions" : [ {
    "ref" : {
      "kind" : "Global",
      "index" : 0
    },
    "blocks" : [ {
      "instructions" : [ {
        "kind" : "LLVMRet",
        "typeId" : 3,
        "operands" : [ {
          "kind" : "Global",
          "index" : 1
        } ],
        "sourceRef" : 0
      } ]
    } ],
    "entryBlockIndex" : 0
  } ],
  "types" : [ {
    "kind" : "LLVMPointerTypeKind",
    "types" : [ 1 ]
  }, {
    "kind" : "LLVMFunctionTypeKind",
    "types" : [ 2 ]
  }, {
    "kind" : "LLVMIntegerTypeKind",
    "len" : 32
  }, {
    "kind" : "LLVMVoidTypeKind"
  } ],
  "globals" : [ {
    "kind" : "LLVMFunctionValueKind",
    "name" : "_ZN12_GLOBAL__N_17trivialEv",
    "typeId" : 0
  }, {
    "kind" : "LLVMConstantIntValueKind",
    "typeId" : 2,
    "intValue" : 1
  } ],
  "sourceRefs" : [ {
    "file" : "parser/anonymousNamespace.cpp",
    "line" : 4
  } ]
}